app/helpers/fuzzy_time.rb

```diff
@@ -22,9 +23,9 @@ def ago(timestamp)
                      when less_than(23.5 * hours)
                        "about #{(diff / (1 * hours)).round} hours ago"
                      end
-          "<span data-toggle='tooltip' title='#{timestamp.strftime('%e %B %Y at %H:%M %Z')}'>#{response}</span>"
+          "<span data-toggle='tooltip' data-title='#{formatted_date}'>#{response}</span>"
         else
-          timestamp.strftime("%e %B %Y at %H:%M %Z")
+          "<span class='localize-time'>#{formatted_date}</span>"
         end
```

Comment:
The `diff` conditional is still relevant. This revision localizes the tooltip for anything under `23.5` hours. The main text (for anything under `23.5` hours) still shows relative time.

![exercism](https://cloud.githubusercontent.com/assets/4740018/17635962/913292b0-6097-11e6-901a-0750be9bf0b1.png)